src/components/FormConfigurator.tsx

```diff
@@ -106,10 +107,11 @@ const FormConfigurator = ({
   const { query, replace } = useRouter();
 
   const [isImported, setIsImported] = useState(false);
+  const [configureRpcRefetchInterval, setConfigureRpcRefetchInterval] = useState(rpcRefetchInterval !== undefined);
 
   const onSelect = useCallback(
     (index: number) => {
-      console.log('called')
+      console.log('called');
```

Comment:
auto formatter

Reply:
imo we can remove these console logs though, doesnt seem necessary to me to leave it there